setup.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+#
+# Copyright 2017-2023 Vrije Universiteit Brussel
+# All rights reserved.
+#
+# This file is part of eb_hooks,
+# originally created by the HPC team of Vrije Universiteit Brussel (https://hpc.vub.be),
+# with support of Vrije Universiteit Brussel (https://www.vub.be),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+##
+"""
+eb_hooks base distribution setup.py
+
+@author: Alex Domingo (Vrije Universiteit Brussel)
+@author: Samuel Moors (Vrije Universiteit Brussel)
+"""
+
+from pathlib import Path
+import setuptools
+
+this_directory = Path(__file__).parent
+long_description = (this_directory / "README.md").read_text()
+
+PKG = {}
+with open("src/eb_hooks/package.py", encoding='utf-8') as fh:
+    exec(fh.read(), PKG)  # pylint: disable=exec-used
+
+setuptools.setup(
```

Comment:
can you make this use vsc-install instead?

Reply:
Our hooks can be of interest to people beyond the VSC. So I suggest to keep this out of the vsc framework to make it easier to use.

Reply:
what we could actually do is move to a `project.toml` instead of the old `setup.py`

Reply:
this will not be installed with rpms but just in the 001 account with a simple `pip install`, in the same way that it is now done for HPC_build_tools

Reply:
we've decided to use vsc-install.
this way we can install locally now and later install system-wide if we want to.